### PR TITLE
Add ACM certificates endpoint to Prism

### PR DESCRIPTION
**Pull request by sihil**
Expiring ACM certificates are becoming a problem to it makes sense to have a way of easily identifying certificates that are close to expiring. Adding an endpoint to prism makes this somewhat easier to do at a departmental level.

The new collector needs a couple of new permissions. This has been added to the Prism role CFN template over at https://github.com/guardian/deploy-tools-platform/pull/61. This needs to be merged / rolled out to all of the accounts before this can be deployed.

TODO:
 - [x] Updated prism CFN template rolled out to all accounts

### CODE REVIEW

File: app/collectors/acmCertificates.scala [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    keyAlgorithm = cert.getKeyAlgorithm,
+    signatureAlgorithm = cert.getSignatureAlgorithm,
+    serial = cert.getSerial,
+    domainValidationOptions = cert.getDomainValidationOptions.asScala.toList.map(DomainValidation.fromApiData),
+    renewalStatus = Option(cert.getRenewalSummary).map(RenewalInfo.fromApiData)
+  )
+}
+
+case class AcmCertificate(
+                              arn: String,
```
Comment (adamnfish) [inline app/collectors/acmCertificates.scala]: I think you need to fix your IntelliJ settings so it indents case class arglists properly. This is an example of where IntelliJ's default is incorrect.

Not super important here, just an FYI.

screen shot 2014-12-23 at 15 48 58

File: app/collectors/acmCertificates.scala [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+case class RenewalInfo(renewalStatus: String, domainValidationOptions: List[DomainValidation])
+
+object RenewalInfo {
+  def fromApiData(ri: RenewalSummary): RenewalInfo = {
+    RenewalInfo(ri.getRenewalStatus, ri.getDomainValidationOptions.asScala.map(DomainValidation.fromApiData).toList)
+  }
+}
+
+object AcmCertificate {
+  def fromApiData(cert: CertificateDetail, origin: AmazonOrigin) = AcmCertificate(
```
Comment (adamnfish) [inline app/collectors/acmCertificates.scala]: Why's the `origin` argument here?

**Comment by mchv** [comment]
👍

File: app/collectors/acmCertificates.scala (hunk 2 above)
Comment (sihil) [inline app/collectors/acmCertificates.scala]: It implements the `origin` in `Collector`

**Pull request merged**